src/app/core/user/user-security/user-security.component.ts

```diff
@@ -0,0 +1,104 @@
+import { Component } from "@angular/core";
+import { DynamicComponent } from "../../view/dynamic-components/dynamic-component.decorator";
+import { OnInitDynamicComponent } from "../../view/dynamic-components/on-init-dynamic-component.interface";
+import { FormBuilder, FormControl, Validators } from "@angular/forms";
+import {
+  KeycloakAuthService,
+  Role,
+} from "../../session/auth/keycloak/keycloak-auth.service";
+import { AuthService } from "../../session/auth/auth.service";
+import { User } from "../user";
+import { MatSnackBar } from "@angular/material/snack-bar";
+import { Observable } from "rxjs";
+import { PanelConfig } from "../../entity-components/entity-details/EntityDetailsConfig";
+
+@DynamicComponent("UserSecurity")
+@Component({
+  selector: "app-user-security",
```

Comment:
I am slightly thrown off by the name - security sounds more technical and sophisticated to me than the account credentials form that this actually implements. But I can live with this also :blush:

Reply:
I am happy for any suggestions!

When coming up with the name, I had the following things in mind:

1. We already have a component that is called `UserAccountComponent` (to edit a users own details)
2. This component handles security aspects as it determines who can access the app and what permissions are given